packages/slate-plugins/src/elements/align/types.ts

```diff
@@ -0,0 +1,43 @@
+import { Element } from 'slate';
+import { RenderElementProps } from 'slate-react';
+import { RenderElementOptions } from '../../element';
+
+const LEFT = 'left';
+const CENTER = 'center';
+const RIGHT = 'right';
```

Comment:
`ALIGN_LEFT =  'align_left'` would be more explicit

Reply:
I can do `ALIGN_LEFT = 'left'` because the value is used for the CSS attribute.